### PR TITLE
Forge - Maintenance Windows

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Create new page for maintenance windows on Forge.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

- Fixes https://github.com/timescale/docs/issues/311
- Fixes https://github.com/timescale/docs/issues/331

### CODE REVIEW

File: timescale-cloud/maintenance.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ necessary, and sometimes require a short outage.
 Upgrades are performed as rolling upgrades where completely new server instances
 are built alongside the old ones. When the new instances are up and running they
 are synchornized with the old servers, and a controlled automatic failover is
+are synchronized with the old servers, and a controlled automatic failover is
```
Comment (clarkbw) [inline timescale-cloud/maintenance.md:47]: This looks like a dupe or copy / pasta 😬

File: timescale-forge/maintenance.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Maintenance
+On Timescale Forge, software updates are handled automatically by us, and you do
```
Comment (clarkbw) [inline timescale-forge/maintenance.md:2]:
```suggestion
On Timescale Forge, minor software updates are handled automatically by us, and you do
```

Thinking we want to ensure people don't confuse this with major PG upgrades.

File: timescale-forge/maintenance.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+not need to perform any actions.
+
+Non-critical software updates are applied during a maintenance window that you
+can define to suit your workload.  If we detect a security vulnerability that
+affects you, we might need to perform maintenance outside of the scheduled
+maintenance window.
+
+<highlight type="important">
+After a maintenance update, the DNS name remains the same, but the IP address
+it points to changes.
```
Comment (clarkbw) [inline timescale-forge/maintenance.md:12]:
```suggestion
it points to most likely changes.
```

It almost always changes AFAIK but I don't think that's a guarantee we want to give.

File: timescale-forge/maintenance.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+</highlight>
+
+Upgrades are performed as zero-downtime upgrades during maintenance windows.
+This means that there is no outage of your services during the upgrade. However,
+all connections and transactions in progress during the upgrade are reset.
```
Comment (clarkbw) [inline timescale-forge/maintenance.md:17]: Should we mention something like: "most database connection systems should auto-retry their connection to restore service connections after this reset."?

**Review by clarkbw — CHANGES_REQUESTED** [review]
Mostly suggestions, one typo

File: timescale-cloud/maintenance.md (hunk 1 above)
Comment (Loquacity) [reply timescale-cloud/maintenance.md:47]: nope, typo fix 😉

Comment (Loquacity) [inline timescale-cloud/maintenance.md:47]:
```suggestion
are synchronized with the old servers, and a controlled automatic failover is
```

File: timescale-forge/maintenance.md (hunk 3 above)
Comment (Loquacity) [reply timescale-forge/maintenance.md:12]:
```suggestion
it points to often changes.
```
??

File: timescale-forge/maintenance.md (hunk 4 above)
Comment (Loquacity) [reply timescale-forge/maintenance.md:17]: Sure!

Comment (Loquacity) [inline timescale-forge/maintenance.md:17]:
```suggestion
Upgrades are performed as zero-downtime upgrades during maintenance windows.
This means that there is no outage of your services during the upgrade. However,
all connections and transactions in progress during the upgrade are reset. In most
cases, the database connection is automatically restored after the reset.
```

**Review by clarkbw — APPROVED** [review]

**Pull request merged**